utility/src/main/java/org/oppia/util/parser/HtmlParser.kt

```diff
@@ -45,12 +47,15 @@ class HtmlParser private constructor(
       htmlContent = htmlContent.replace("&amp;quot;", "")
     }
 
-    val imageGetter = urlImageParserFactory.create(htmlContentTextView, entityType, entityId, imageCenterAlign)
+    val imageGetter = urlImageParserFactory.create(
+      htmlContentTextView, entityType, entityId, imageCenterAlign)
```

Comment:
I think it should be like this:
```
val imageGetter = urlImageParserFactory.create(
      htmlContentTextView, entityType, entityId, imageCenterAlign
)
```

Not sure if this is correct from kotlin style guide point of view.